components/server/src/installation-admin/telemetry-data-provider.ts

```diff
@@ -18,9 +18,9 @@
      async getTelemetryData(): Promise<TelemetryData> {
             const data: TelemetryData = {
                 installationAdmin: await this.installationAdminDb.getData(),
-                totalUsers: await this.userDb.getUserCount(true),
-                totalWorkspaces: await this.workspaceDb.getWorkspaceCount(),
-                totalInstances: await this.workspaceDb.getInstanceCount(),
+                totalUsers: 0,  //await this.userDb.getUserCount(true),
```

Comment:
Why disable the user count query? (This seems unrelated to an index on workspace.type)

Reply:
It was Saturday morning - and retrospectively, I would argue to be consistent :wink: